### ISSUE TITLE
special character parse_json_array: invalid character '\x15' in string literal

### ISSUE THREAD

**Issue by IRPsych** [Velocidex/velociraptor#3962]
Guys there is an invalid character when VRaptor run hunts and parsing data to get stdout, this character is (\x15) and will not return any results on any hunt that has it. This will affect analysts jumping to conclude that the hunt went okay and nothing is there.

**Comment by scudette** [comment]
I guess the issue is here that parse_json_array() is emitting the error at INFO level and not ERROR level. Therefore it is hard to know that the collection failed. We should emit this error at the ERROR level since it is probably a serious error.

Error handling is discussed here: https://docs.velociraptor.app/docs/vql/#vql-error-handling 

It is not necessary that the collection be marked as failed when some errors occur - users should always check the query error logs

**Comment by IRPsych** [comment]
Yep not from the collection but from the JSON parser it some how decided to not to return anything.

I ran the same collection against the same data through power-shell and I was able to retrieve what I wanted in json.

**Comment by scudette** [comment]
This is probably a bug with the VQL - can you attach the exact artifact you ran?

Usually when filtering the output of powershell it will be unicode utf16 encoded and needs to be decoded before further processing.

**Comment by IRPsych** [comment]
Sure, here is the artifact details:
Name: Exchange.Windows.ActiveDirectory.PrivilegedUsers

best,

**Comment by IRPsych** [comment]
Thanks a lot for your prompt response. 
That's got to do. 
Cheers,
